roles/pcc-api/README.md

```diff
@@ -0,0 +1,46 @@
+Role Name
+=========
+
+A brief description of the role goes here.
+
+Requirements
+------------
+
```

Comment:
here we put the ansible version and for your role the maria-db version and other versions that should be included.